### PR TITLE
log page views on docs.pterohq.com

### PR DESCRIPTION
**Pull request by cameron**

### CODE REVIEW

File: _templates/layout.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    analytics.load(document.location.hostname == 'docs.pterohq.com'
+    ? "aTK42fHnthz8RDQwlDjIzdYdkd1CYGkj"
+    : "Bksiqv27kRC1fOASWhC7j9fIIgB9YDVv");
+    }}();
+    var id = document.cookie.match(/userId=([^;]*)/);
+    id[1] && analytics.identify(id[1]);
+    analytics.page('docs' + document.location.pathname);
```
Comment (techdubb) [inline _templates/layout.html:13]: Do we need a separator between `docs` and the `pathname`?

Comment (cameron) [inline _templates/layout.html:13]: pathname always starts with '/'

Comment (techdubb) [inline _templates/layout.html:13]: :+1:

**Comment by gooley** [comment]
:chart:

**Pull request merged**